orchestrator/src/handlers/price/mod.rs

```diff
@@ -62,9 +66,6 @@ pub async fn fetch_canister_logs(
         chrono::DateTime::from_timestamp(i64::try_from(state.start_timestamp)?, 0).unwrap();
     debug!("Fetching canister logs since {:?}", start_timestamp);
```

Comment:
The debug message looks weird here, and the `start_timestamp` var is only used for the debug message.